CHANGES

```diff
@@ -9,6 +9,15 @@
 
  Changes between 1.1.1 and 3.0.0 [xx XXX xxxx]
 
+  *) Early start up entropy quality from the DEVRANDOM seed source has been
+     improved for older Linux systems.  The RAND subsystem will wait for
+     /dev/random to be producing output before seeding from /dev/urandom.
+     The seeded state is stored for future library initialisations using
+     a system global shared memory segment.  The shared memory identifier
+     can be configured by defining OSSL_SHM_SEEDED to the desired value.
```

Comment:
Hmm, when we are already at nit-picking: this is the wrong define, it is actually
OPENSSL_RAND_SEED_DEVRANDOM_SHM_ID

Reply:
Fixing in new PR.